### ISSUE TITLE
Regex format (case insensitive /i)

### ISSUE THREAD

**Issue by businessBoris** [Quicksaver/Tab-Groups#518]
More a request / information than an issue.

What format does the regex parser want in the tab-grab?
I've tried `/.*CoreOS.*/i` to drop case sensitivity.
May I beg for a wiki page or two or an answer and I'll create one.

Out of this world add-on BTW. Will buy hours in new year.

**Comment by Quicksaver** [comment]
Actually you can't, that was an oversight on my part, I'm sorry about that. Right now, in your example, you only need to use (and can only use) ```.*CoreOS.*``` (no forward slashes or any flags), although I would use ```.*coreos.*``` if that's a domain, as domain names should be case insensitive and Firefox already converts them to lowercase automatically.

I'll try to find a way to add a way to make it case insensitive for the next update, shouldn't be too hard. :)

**Comment by businessBoris** [comment]
Thank you. I tend to create most of my tabs from my phone, so having 99 tabs about donkey ballet in my coreos tab is not too helpful; I find it too distracting! ;-)

If I read between the lines, you're searching domains only, not titles or URLs?
URLs would actually helpful too; if two(+) rules conflict/match it doesn't really matter if one or two tabs are in the 'wrong' place.  The search is SOOOOOOO helpful! - I tend to use it in preference to the main bar.

**Comment by Quicksaver** [comment]
> If I read between the lines, you're searching domains only, not titles or URLs?

No, you can filter using the full url. I meant that, in the W3C specs, the domain of a website is case insensitive (e.g. coreos.com is the same as CoReOs.CoM), so Firefox automatically lowercases that bit regardless of how you type it in the location bar. The rest of the url can be case sensitive, depending on the server software running the website, so I believe Firefox doesn't lowercase the rest of the url, only the domain bit. You can still use rules for whole urls, but you need to keep a close eye on the case of the url as similar but upper and lower case urls can lead to different pages.

**Comment by businessBoris** [comment]
Thank you! Just filling in a few now. There're "\b" everywhere (git not github...).

**Comment by WindfallProphet** [comment]
Don't mean to intrude, but is there an easy way to grab all the tabs that match a string (I guess we are using $donkey_ballet this time)? I didn't see it written in the help section.

**Comment by Quicksaver** [comment]
@WindfallProphet I really don't understand your question.
